src/main/java/com/google/alpollo/ConfigHelper.java

```diff
@@ -0,0 +1,41 @@
+package com.google.alpollo;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonIOException;
+import com.google.gson.JsonSyntaxException;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.Objects;
+
+/**
+ * Helps to load configuration file and retrieve data from it.
+ */
+final public class ConfigHelper {
+  private static final String configFileName = "config.json";
+  private static final Gson gson = new Gson();
+
+  /**
+   * Class which represents the content of configuration file,
+   * used only to get Json object from file.
+   */
+  private static final class ConfigInfo {
+    private String projectID;
+
+    public String getProjectID() {
+      return projectID;
+    }
+  }
+
+  private ConfigHelper() {
+    throw new RuntimeException("Instantiation of ConfigHelper is not allowed!");
+  }
+
+  public static String getProjectID() throws NullPointerException,
```

Comment:
You don't need to declare `RuntimeException`s, e.g. `NullPointerException`. They fail at run time, so there is no need to tell the compiler about them.

Also, you should not simply propagate these JSON exceptions, otherwise all the callers will need to declare/import them as well. Instead, you can have a `try/catch` here and do one of the following when catching them:

- Throw another exception with a relevant message, e.g. "Failed to obtain Project ID.". The exception thrown can be an `IllegalStateException` or another one from the Java standard library that you think is more appropriate. The caller can then use a try/catch or rethrow the Exception without needing to import custom exceptions, e.g. `JsonIOException`.
- Return null. In this case the caller will check if the `projectId` is null and respond accordingly.